src/cls/IPM/Storage/UpdateStep.cls

```diff
@@ -39,6 +39,30 @@ Method %OnNew(
     return $$$OK
 }
 
+/// Returns %IPM.Storage.UpdateStep object corresponding to unique index in storage table
+/// If object does not exist, this will return an empty object
+ClassMethod GetUpdateStepObject(
+	className As %String,
+	methodName As %String,
+	mirrored As %Boolean,
+	mirroredHostName As %String,
+	mirroredInstanceName As %String) As %IPM.Storage.UpdateStep
+{
+    set tRes = ##class(%SQL.Statement).%ExecDirect(,
+        "SELECT ID "_
+        "FROM %IPM_Storage.UpdateStep "_
+        "WHERE ClassName = '"_className_
+        "' AND MethodName = '"_methodName_
+        "' AND MirroredHostName = '"_mirroredHostName_
+        "' AND MirroredInstanceName = '"_mirroredInstanceName_"'")
```

Comment:
Nit: if there is a unique index, can use the generated `<index name>Exists` method instead as its more performant (no time spent to prepare the query)